data-generation/src/main/scala/thesis/SnapshotDelta.scala

```diff
@@ -27,10 +27,10 @@ abstract class TemporalGraph[VD: ClassTag, ED: ClassTag] extends Serializable {
 
 class SnapshotDelta(val graphs: MutableList[Snapshot],
                     val logs: RDD[LogTSV],
-                    val snapshotType: SnapshotIntervalType) extends TemporalGraph[Attributes, Attributes] { // extends Graph[VD, ED] {
+                    val snapshotType: SnapshotIntervalType) extends TemporalGraph[Attributes, (EdgeId, Attributes)] { // extends Graph[VD, ED] {
   override val vertices: VertexRDD[Attributes] = graphs.get(0).get.graph.vertices
-  override val edges: EdgeRDD[Attributes] = graphs.get(0).get.graph.edges
-  override val triplets: RDD[EdgeTriplet[Attributes, Attributes]] = graphs.get(0).get.graph.triplets
+  override val edges: EdgeRDD[(EdgeId, Attributes)] = graphs.get(0).get.graph.edges
```

Comment:
Hmm, kanskje man skulle laget enda en case class. Så man får ` case class EdgePayload(id:Long, attributes:Attributes)`
Da blir det `edge.attr.attributes`for å hente ut attributes lol

Reply:
Tenker det er greit nok sånn her når man kun har Id og Attributter, men hvis du gidder så kan det være smooth med det andre og